src/main/app/fees/v2/forms/model/CreateFeeVersionForm.ts

```diff
@@ -9,42 +9,42 @@ import {
 } from 'fees/v2/model/fees-register-api-contract'
 
 export class CreateFeeVersionForm {
-  @IsDefined ( { message: ValidationErrors.TYPE_REQUIRED } )
-  @IsNotBlank ( { message: ValidationErrors.TYPE_REQUIRED } )
+  @IsDefined({ message: ValidationErrors.TYPE_REQUIRED })
+  @IsNotBlank({ message: ValidationErrors.TYPE_REQUIRED })
   amountType?: string
 
-  @IsDefined ( { message: ValidationErrors.DESCRIPTION_REQUIRED } )
-  @IsNotBlank ( { message: ValidationErrors.DESCRIPTION_REQUIRED } )
-  @MaxLength ( 2000, { message: ValidationErrors.DESCRIPTION_TOO_LONG } )
+  @IsDefined({ message: ValidationErrors.DESCRIPTION_REQUIRED })
+  @IsNotBlank({ message: ValidationErrors.DESCRIPTION_REQUIRED })
+  @MaxLength(2000, { message: ValidationErrors.DESCRIPTION_TOO_LONG })
```

Comment:
For easy review, I would separate bulk of these format changes into a different PR.